act-rules-format.bs

```diff
@@ -451,63 +496,67 @@ Definitions {#definitions}
         <li>An EPUB document</li>
         <li>A Vue component file</li>
       </ul>
     </div>
     <div class=note>
-      <p><strong>Note</strong>: Implementers using the [[EARL10-Schema]] can express the test subject with the [subject property](https://www.w3.org/TR/EARL10-Schema/#subject)</p>
+      <p>**Note:**>: Implementers using the [[EARL10-Schema]] can express the test subject with the [subject property](https://www.w3.org/TR/EARL10-Schema/#subject)</p>
     </div>
   </dd>
 
   <dt><dfn>Test Target</dfn></dt>
   <dd>
-    <p>A distinct part of the [=test subject=].</p>
+    <p>A distinct part of the [=test subject=], as defined by the [applicability](#applicability).</p>
     <div class=example>
-      <p><strong>Example:</strong></p>
+      <p>**Example:**</p>
       <ul>
         <li>Nodes within an HTML page</li>
         <li>A period of time within a video</li>
         <li>A range of characters within a text document</li>
       </ul>
     </div>
     <div class=note>
-      <p><strong>Note</strong>: Implementers using the [[EARL10-Schema]] can express the test target with the [pointer property](https://www.w3.org/TR/EARL10-Schema/#pointer)</p>
-    </div>
-  </dd>
-
-  <dt><dfn>Outcome</dfn></dt>
-  <dd>
-    <p>One of three types of conclusions that come from evaluating an ACT Rule on a [=test subject=]. An outcome can be:<p>
-    <ul>
-      <li>**Inapplicable**: No part of the test subject matches the applicability</li>
-      <li>**Passed**: The test subject meets all expectations</li>
-      <li>**Failed**: The test subject does not meet all expectations</li>
-    </ul>
-    <div class="note">
-      <p><strong>Note</strong>: While *inapplicable* is a valid outcome for ACT Rules, it might not be a valid result for all [=accessibility requirements=]. Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to satisfied (passed and inapplicable) or not satisfied (failed).</p>
-    </div>
-    <div class="note">
-      <p><strong>Note</strong>: Implementers using the [[EARL10-Schema]] can express the outcome with the [outcome property](https://www.w3.org/TR/EARL10-Schema/#outcome). In addition to `passed` `failed` and `inapplicable`, EARL 1.0 also defined an `incomplete` outcome. While this can not be the outcome of an ACT Rule when applied in its entirety, it often happens that rules are only partially evaluated. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the `incomplete` outcome.
+      <p>**Note:** Implementers using the [[EARL10-Schema]] can express the test target with the [pointer property](https://www.w3.org/TR/EARL10-Schema/#pointer)</p>
     </div>
   </dd>
 </dl>
 
+
 Appendix 1: Expressing ACT Rule results with JSON-LD and EARL {#appendix-data-example}
 =================================================================================
```

Comment:
**Editorial**:

- We need to put the context somewhere on w3.org
- Example 2 is incorrect, it should not be one result with two pointers. This has to have two results
- @nitedog Can you confirm that the "outcome" has to point to the instance ("failed") rather than to the class ("Fail") in EARL? - in either case, "Failed" capital first is invalid EARL

Reply:
Use correct markup so examples are numbered.

Example 2 should change to:
{
  "@context": "https://raw.githubusercontent.com/w3c/wcag-act/master/earl-act.json",
  "@graph": [{
    "@type": "Assertion",
    "subject": "https://example.org/",
    "test": "auto-wcag:SC1-1-1-css-image.html",
    "result": {
      "outcome": "earl:failed",
      "pointer": "html > body > h1:first-child"
    }
  }, {
    "@type": "Assertion",
    "subject": "https://example.org/",
    "test": "auto-wcag:SC1-1-1-css-image.html",
    "result": {
      "outcome": "earl:passed",
      "pointer": "html > body > h1:nth-child(2)"
    }
  }]
}


Example 3 needs changing to:
{
  "@context": "https://raw.githubusercontent.com/w3c/wcag-act/master/earl-act.json",
  "@type": "Assertion",
  "subject": {
    "@type": ["WebSite", "TestSubject"],
    "@value": "https://example.org/"
  }
  "test": "http://www.w3.org/WAI/WCAG2A-Conformance",
  "result": {
    "outcome": "earl:failed",
    "source": [{
      "test": "wcag20:text-equiv-all",
      "result": {
        "outcome": "earl:failed",
        "source": [ … ]
      }
    }, {
      "test": "wcag20:media-equiv-av-only-alt",
      "result": {
        "outcome": "earl:passed",
        "source": [ … ]
      }
    }, {
      "test": "wcag20:media-equiv-captions",
      "result" : {
        "outcome": "earl:inapplicable",
        "source": [ … ]
      }
    }, … ]
  }
}